frontend/src/ClassDetail.svelte

```diff
@@ -1,275 +1,429 @@
 <script>
-  import {link} from 'svelte-spa-router'
-  import {fetch} from './api.js'
+  import { link } from "svelte-spa-router";
+  import { fetch } from "./api.js";
```

Comment:
I think you used some kind of automatic formatter and it created a lot of changes in this file. Can you just "undo" these changes, so we can easily see what has really changed?

Then just commit it again with: `git commit --amend` and the force push this commit to this branch.

Thank you.